src/renderer/wg_engine/tvgWgPipelines.h

```diff
@@ -65,7 +65,8 @@ class WgPipelines {
     WGPURenderPipeline radial[2]{};
     WGPURenderPipeline linear[2]{};
     WGPURenderPipeline image[2]{};
-    WGPURenderPipeline sceneComp[12];
+    WGPURenderPipeline sceneClip;
+    WGPURenderPipeline sceneComp[12]; // TODO: update to 11 after removing CompositeMethod::ClipPath enum value
```

Comment:
I can't imagine the reason this. I think it would be better to remove it altogether.

Reply:
@hermet 
> I can't think of a reason for this. The `Composite::ClipPath` command is never requested from the pipeline. it would be better to remove it altogether.

It was requested, but it was hiden in the routines. All other composite methods must be it this array, because pipelines request by composite method as an index.

Reply:
@SergeyLebedkin  please use `[idx - 1]` no need to remove dead resource.